code/__DEFINES/~skyrat_defines/DNA.dm

```diff
@@ -58,7 +58,11 @@
 #define PREVIEW_PREF_LOADOUT "Loadout"
 #define PREVIEW_PREF_NAKED "Naked"
 
-#define MANDATORY_FEATURE_LIST list("mcolor" = "FFB","mcolor2" = "FFB","mcolor3" = "FFB","ethcolor" = "FCC","skin_color" = "FED","flavor_text" = "","breasts_size" = 1,"breasts_lactation" = FALSE,"penis_size" = 13,"penis_girth" = 9,"penis_taur_mode" = TRUE,"balls_size" = 1)
+#define BODY_SIZE_NORMAL 		1.00
+#define BODY_SIZE_MAX			1.25
```

Comment:
Didn't we have a max of 1.50 before?

Reply:
We used a config. I personally don't like over-the-top sizes as they break up the boundaries way too much, and distort pixels, all while making clicking on things difficult. I think they work great as an indication if someone is bigger/smaller than usual, which these values do pretty well.